packages/website/src/styles/main.css

```diff
@@ -1,10 +1,5 @@
 @import './_syntax-highlighting.css';
 
-body {
```

Comment:
is this unnecessary because we set it in global.css ?

Reply:
yes exactly, forgot to remove it in my PR for global.css